src/cookbooks/bigquery.md

```diff
@@ -163,37 +166,84 @@ Note: this is very similar to [API Access](bigquery.md#gcp-bigquery-api-access),
+but it has changed from `YYYYMMDD` string form to a proper `DATE` type that accepts string literals in the more standards-friendly `YYYY-MM-DD` form.
 - Unqualified queries can become very costly very easily. We've placed restrictions on large tables from accidentally querying "all data for all time",
 namely that you must make use of the date partition fields for large tables (like `main_summary` or `clients_daily`).
 - Please read [_Query Optimizations_](bigquery.md#query-optimizations) section that contains advice on how to reduce cost and improve query performance.
 - re:dash BigQuery data sources will have a 10 TB data scanned limit per query. Please let us know in `#fx-metrics` on Slack if you run into issues!
-- There are no other partitioning fields in BigQuery versions of parquet datasets (e.g. `sample_id` is no longer a partitioning field and will not necessarily reduce data scanned).
 - There is no native map support in BigQuery. Instead, we are using structs with fields [key, value]. We have provided convenience functions to access these like key-value maps (described [below](bigquery.md#accessing-map-like-fields).)
 
 ### Projects with BigQuery datasets
 
 |Project|Dataset|Purpose|
 |---|---|---|
-|`moz-fx-data-derived-datasets`|    |Imported parquet data, [BigQuery ETL](https://github.com/mozilla/bigquery-etl), ad-hoc analysis|
+|`moz-fx-data-shared-prod` |    |All production data including full pings, imported parquet data, [BigQuery ETL](https://github.com/mozilla/bigquery-etl), and ad-hoc analysis |
+|   |`<namespace>_live`    |See _live datasets_ below|
+|   |`<namespace>_stable`  |See _stable datasets_ below|
+|   |`<namespace>_derived` |See _derived datasets_ below|
+|   |`<namespace>`         |See _user-facing (unsuffixed) datasets_ below|
 |   |`analysis`|User generated tables for analysis|
 |   |`backfill`|Temporary staging area for back-fills|
 |   |`blpadi`|Blocklist ping derived data(_restricted_)|
+|   |`payload_bytes_raw` |Raw JSON payloads as received from clients, used for reprocessing scenarios, a.k.a. "landfill" (_restricted_) |
+|   |`payload_bytes_decoded` |`gzip`-compressed decoded JSON payloads, used for reprocessing scenarios  |
+|   |`payload_bytes_error` |`gzip`-compressed JSON payloads that were rejected in some phase of the pipeline; particularly useful for investigating schema validation errors|
 |   |`search`|Search data imported form parquet (_restricted_)|
-|   |`static`|Static data for use with analysis|
-|   |`telemetry`|User-facing views on imported parquet data and tables generated from [BigQuery ETL](https://github.com/mozilla/bigquery-etl)     |
-|   |`telemetry_raw`|Imported parquet data|
+|   |`static`|Static tables, often useful for data-enriching joins|
 |   |`tmp`|Temporary staging area for parquet data loads|
+|   |`udf` |Persistent user-defined functions defined in SQL|
+|   |`udf_js` |Persistent user-defined functions defined in JavaScript|
```

Comment:
Can this link to https://github.com/mozilla/bigquery-etl/tree/master/udf_js or a UDF documentation page?

Reply:
I opened #312 for the long term solution but I think these links would be useful in the interim.